Demo/src/main/java/com/paypal/android/ui/card/CardViewModel.kt

```diff
@@ -4,9 +4,22 @@ import android.util.Log
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.paypal.android.card.Card
+import com.paypal.android.card.CardClient
+import com.paypal.android.core.Environment
+import com.paypal.android.core.PaymentsConfiguration
 
 class CardViewModel : ViewModel() {
 
+    companion object {
+        private val TAG = CardViewModel::class.qualifiedName
+        private const val ORDER_ID = "45413380SJ116311E"
+        private const val CLIENT_ID =
+            "ASUApeBhpz9-IhrBRpHbBfVBklK4XOr1lvZdgu1UlSK0OvoJut6R-zPUP7iufxso55Yvyl6IZYV3yr0g"
+        private const val AUTH_TOKEN =
+            "A21AAKm8H7XlpB0LWC6hHDbEKOzn5bZzSbt74-QCOYl63WnnHWjKwNttWZyCPiY1tfRsB5PFWV6iBmmulC01Y5BleYXNlRGLw"
```

Comment:
I'm not sure if we should be checking in ClientIds or auth tokens. Can we remove these and add a placeholder?

Reply:
Yeah this was a hack but the auth token at least has an expiry. I agree we could inject `CLIENT_ID` and `AUTH_TOKEN` via a project-level, .gitignore'd `gradle.properties`.

Reply:
`paypal.properties` file can be used to load CLIENT_ID and CLIENT_SECRET without having to check them into the repo.